doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -798,9 +798,17 @@ and the terminating expression is ``nil``. Here are other examples:
            (pair .. (pair (pair a u) (pair b u)) .. (pair c u)))
      (t at level 39).
 
-Notations with recursive patterns can be reserved like standard
-notations, they can also be declared within
-:ref:`notation scopes <Scopes>`.
+   Notation "x  ⪯ y  ⪯ ..  ⪯ z  ⪯ t" :=
+     ((fun b A a => a <= b /\ A b) y .. ((fun b A a => a <= b /\ A b) z (fun b => b <= t)) .. x)
+     (at level 70, y at next level, z at next level, t at next level).
+
+even if the latter, a bit tricky, is also fragile, as it introduces a
+beta-redex which tactics are liable to reduce, thus breaking the
+printing of the notation.
```

Comment:
The printing breakage is that `x  ⪯ y  ⪯ z` may beta reduced and then not be printed exactly the same way?  If so, how about stating that?

Reply:
> The printing breakage is that `x ⪯ y ⪯ z` may beta reduced and then not be printed exactly the same way?

Yes.

> If so, how about stating that?

Ideally, this kind of notation should be provided natively, i.e. without having to use a "hack". But that'd be a bit of work... 

Maybe the simplest would be to add a comment before the example telling something like: "As yet another example, we show a way to write chains of inequalities. This is not entirely satisfactory as it relies on a tricky encoding but it gives a flavor of the strength (and limits) of recursive notations."?